addons/functions/functions/fnc_initTrench.sqf

```diff
@@ -25,11 +25,14 @@ if (is3DEN) exitWith {
 
 if (isServer) then {
    _object setVariable ["ace_trenches_progress", 1, true];
-   _object setVariable ["ace_trenches_placeData", [_pos, _vecDirAndUp], true];
 
     if (GVAR(allowTrenchDecay)) then {
        [_object, GVAR(timeoutToDecay), GVAR(decayTime)] call FUNC(decayPFH);
     };
+
+    if (GVAR(allowHitDecay)) then {
+       [_object, GVAR(hitDecayMultiplier)] remoteExec [QFUNC(hitEH), 0, true];
```

Comment:
Could we use the CBA System instead of remoteExec so that the code works when remoteExec is blocked.